.puppet-lint.rc

```diff
@@ -1,3 +1 @@
 --fail-on-warnings
---no-parameter_documentation-check
```

Comment:
Please also configure it in .sync.yml, otherwise modulesync will readd it.

Reply:
will do